disk/datadog_checks/disk/disk.py

```diff
@@ -20,13 +20,42 @@
     # See: https://github.com/DataDog/integrations-core/pull/1109#discussion_r167133580
     IGNORE_CASE = re.I
 
+    KNOWN_SPECIAL_FILE_SYSTEMS = set()
+
     def _base_device_name(device):
         return device.strip('\\').lower()
 
 
+elif platform.system() == "Linux":
+    IGNORE_CASE = 0
+
+    # Remember to update the configuration specification
+    # if modifying the list of special file systems.
+    KNOWN_SPECIAL_FILE_SYSTEMS = set(
+        [
+            'binfmt_msc',
```

Comment:
There's a mismatch between the spec and here, I guess this is supposed to be [`binfmt_misc`](https://en.wikipedia.org/wiki/Binfmt_misc)?

Reply:
Indeed, thanks for catching that!